.gitignore

```diff
@@ -56,3 +56,6 @@ codegen
 
 **/cspell.json
 !./cspell.json
+
+mutants.out
+mutants.out.old
```

Comment:
mutants?

Reply:
Output directories for https://github.com/sourcefrog/cargo-mutants. Didn't seem like much harm in adding them here